test/preprocessors/MongoidPreprocessor.test.ts

```diff
@@ -67,8 +85,14 @@ describe("MongoidPreprocessor", () => {
     expect(entities[0].collectionName).toBeNull();
     expect(entities[1].collectionName).toBe("fhir_measures");
 
+    expect(entities[2].dataType.typeName).toBe("Resource");
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)